### ISSUE TITLE
0.54.0 & empty imported file => import loop (false) detected

### ISSUE THREAD

**Issue by thybzi** [stylus/stylus#2142]
Stylus 0.54.0 introduced a weird bug when importing empty file

Minimal setup to reproduce:
```
first.styl    // @import 'second'
              // @import 'third'
second.styl   // (empty)
third.styl    // @import 'second'
```

```stylus first.styl``` reports:
```
Error: third.styl:1:9
   1| @import 'second'
--------------^
   2|

import loop has been found
```

If I swap import lines order in `first.styl` (`third`, then `second`), bug doesn't reproduce.
If I add just any content to `second.styl` (may it be even empty comment `//`), bug doesn't reproduce.
If I remove or comment out import line in `third.styl`, bug doesn't reproduce.
If I downgrade to Stylus 0.53.0, bug doesn't reproduce.

**Comment by thybzi** [comment]
Huh! Thanks for quick fix! :+1:  
Waiting for version update :)

**Comment by Panya** [comment]
Fixed in `0.54.1`
